source/nodejs/adaptivecards-designer/src/containers/berlin/berlin-container.ts

```diff
@@ -1,34 +1,42 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import * as Adaptive from "adaptivecards";
-import { HostContainer, ColorTheme } from "../host-container";
+import { ThemeSupportedHostContainer } from "../theme-supported-host-container";
 import * as hostConfigLight from "../../hostConfigs/berlin-light.json";
 import * as hostConfigDark from "../../hostConfigs/berlin-dark.json";
 
 export enum ContainerSize {
     Small = "Small",
     Medium = "Medium",
     Large = "Large"
-};
+}
 
-export class BerlinContainer extends HostContainer {
+export class BerlinContainer extends ThemeSupportedHostContainer {
     private _containerSize: ContainerSize;
-    private _colorTheme: ColorTheme;
 
-    constructor(size: ContainerSize, theme: ColorTheme) {
-        super("Berlin (Test)",
-            `containers/berlin-container-${theme.toLowerCase()}.css`);
+    constructor(size: ContainerSize) {
+        super(
+            "Berlin (Test)",
+            "berlin-container",
+            hostConfigLight,
+            hostConfigDark,
+            "#D2D2D2",
+            "#616161"
+        );
         this._containerSize = size;
-        this._colorTheme = theme;
     }
 
     public initialize(): void {
         super.initialize();
-        Adaptive.GlobalSettings.removePaddingFromContainersWithBackgroundImage = true; 
+        Adaptive.GlobalSettings.removePaddingFromContainersWithBackgroundImage = true;
     }
 
     public renderTo(hostElement: HTMLElement) {
-        this.cardHost.classList.remove("berlin-small-card", "berlin-medium-card", "berlin-large-card");
+        this.cardHost.classList.remove(
```

Comment:
minor (no action required): I would avoid hard code like this because berlin-small-card name is generated by code.

For example, it's easy to introduce bug if remove `.toLowerCase` someday but forget the make change on this.

A good solution is:
```
type ContainerSizeType = "small"|"medium"|"large";
private getCardName(type: ContainerSizeType): string;
const ValidContainerSize:   ContainerSizeType[] = ["small", "medium", "large"];
```

Reply:
This is a great call out - I'll keep this in mind in the future!